clang/test/Driver/aarch64-bf16.c

```diff
@@ -5,4 +5,4 @@
 // RUN: %clang --target=aarch64 -march=armv8.5a+bf16+nobf16 -### -c %s 2>&1 | FileCheck -check-prefix=GENERICV85A-BF16-NO-BF16 %s
 // GENERICV85A-BF16-NO-BF16: "-target-feature" "-bf16"
 // RUN: %clang --target=aarch64 -march=armv8.5a+bf16+sve -### -c %s 2>&1 | FileCheck -check-prefixes=GENERICV85A-BF16-SVE %s
-// GENERICV85A-BF16-SVE: "-target-feature" "+bf16" "-target-feature" "+sve"
+// GENERICV85A-BF16-SVE: "-target-feature" "+bf16"{{.*}} "-target-feature" "+sve"
```

Comment:
In general, why are these `{{.*}}` now needed in many tests? Is it that we're listing features we didn't before, because before they were expanded in the backend?

Features that are unrelated to, in this case, bf16.

Reply:
Yes, we now list every optional feature which the architecture/cpu has by default, most of which aren't relevant to the tests. The order has also changed, so that it matches the source-code order in AArch64TargetParser.h

Reply:
Understood.